mathics/builtin/base.py

```diff
@@ -844,25 +844,33 @@ def error(self, tag, *args):
     def error_args(self, count, *expected):
         raise PatternArgumentError(self.get_name(), count, *expected)
 
-    def get_lookup_name(self) -> str:
-        return self.get_name()
+    def get_attributes(self, definitions) -> int:
+        """
+        If self has head return head's attributes.
+        Otherwise that no attributes have been set.
+        """
+        if self.head is None:
+            # FIXME: _Blank in builtin/patterns.py sets head to None.
+            # Figure out if this is the best thing to do and explain why.
+            return nothing
```

Comment:
@TiagoCavalcante it really bothers me what we have an attribute called `nothing`.  Please consider changing it to `no_attributes`. Thanks.

Reply:
no_attributes seems better, I'll change it after merging this PR.